src/service/chat.js

```diff
@@ -245,19 +255,18 @@ const getAllowAnonDmStatus = async (postId) => {
   }
 };
 
-const initChatFromPostAnon = async ({source, id}) => {
+const initChatFromPostAnon = async ({source, id, feedId}) => {
   let payload = {source};
   switch (source) {
     case 'post':
       payload = {...payload, postId: id};
       break;
     case 'comment':
-      payload = {...payload, commentId: id};
+      payload = {...payload, commentId: id, postId: feedId};
       break;
     default:
       break;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the `initChatFromPost` function, `initChatFromPostAnon` also includes a new parameter `feedId`. The logic seems fine and no issues are found.


<!-- This is an auto-generated comment by OSS CodeRabbit -->